common/lib/client/auth.js

```diff
@@ -534,6 +519,144 @@ var Auth = (function() {
 		}
 	};
 
+	Auth.prototype._saveBasicOptions = function(defaultTokenParams, authOptions) {
+		this.method = 'basic';
+		this.key = authOptions.key;
+		this.basicKey = toBase64(authOptions.key);
+		this.tokenParams = defaultTokenParams || {};
+		this.authOptions = authOptions || {};
+		delete this.authOptions.force;
```

Comment:
I don't think we should use `delete` - why not `this.authOptions.force = false` ?